api/files/views.py

```diff
@@ -320,6 +320,9 @@ def get_node(self):
 
     # overrides RetrieveAPIView
     def get_object(self):
+        if self.request.GET and self.request.GET.get('giveGuid', False):
```

Comment:
Prefer `underscore_method` to `camelCase` for variables, as specified in the [dev docs](http://cosdev.readthedocs.io/en/latest/style_guides/python.html)